tidy3d/plugins/dispersion/fit_web.py

```diff
@@ -122,31 +196,25 @@ def fit(  # pylint:disable=arguments-differ, too-many-arguments, too-many-locals
             Number of optimizations to run with random initial guess.
         tolerance_rms : NonNegativeFloat, optional
             RMS error below which the fit is successful and the result is returned.
-        bound_amp : PositiveFloat, optional
-            Bound on the amplitude of poles, namely on Re[c], Im[c], Re[a]
-        bound_f : PositiveFloat, optional
-            Bound on the frequency of poles, namely on Im[a]. Unit: Hz
-        constraint : Literal["hard", "soft"], optional
-            'hard' or 'soft'
-        nlopt_maxeval : PositiveInt, optional
-            Maxeval in each optimization
+        advanced_param : AdvancedFitterParam, optional
+            Other advanced parameters.
```

Comment:
let's format this like 
```
:class:`AdvancedFitterParam`
```
so it links in the docs.

Reply:
All revised now.